rasa/shared/core/trackers.py

```diff
@@ -475,10 +480,21 @@ def _define_user_featurization(future_events: List[Event]) -> bool:
             elif isinstance(future_event, DefinePrevUserUtteredFeaturization):
                 return future_event.use_text_for_featurization
 
+    @staticmethod
+    def _define_user_entities(
+        future_events: List[Event],
+    ) -> Optional[List[Dict[Text, Any]]]:
+        for future_event in future_events:
+            if isinstance(future_event, DefinePrevUserUtteredEntities):
+                return future_event.entities
```

Comment:
we have to search only until next `AtionExecuted` because the search should happen only within one dialogue turn. See `_define_user_featurization` for reference